lms/resources/_js_config/__init__.py

```diff
@@ -198,6 +198,14 @@ def enable_lti_launch_mode(self, assignment: Assignment):
         }
         self._config["debug"]["values"] = self._get_lti_launch_debug_values()
 
+        self._config["editing"] = {
+            # Endpoint to get any data needed to get into "editing mode"
+            "getConfig": {
+                "path": self._request.route_path("lti.reconfigure"),
```

Comment:
Send the details of the new "API" to the FE.

The `data` happens to be our LTIParams.